cmd/access/main.go

```diff
@@ -75,10 +75,11 @@ func main() {
 		logTxTimeToFinalizedExecuted bool
 		retryEnabled                 bool
 		rpcMetricsEnabled            bool
+		stakedAccessNodeAddress      string
 	)
 
 	cmd.FlowNode(flow.RoleAccess.String()).
-		ExtraFlags(func(flags *pflag.FlagSet) {
+		ExtraFlagsWithFnb(func(flags *pflag.FlagSet, fnb *cmd.FlowNodeBuilder) {
```

Comment:
The `unstaked` command line arg determines if the AN starts as a staked (unstaked=false) or an unstaked tunstaked=rue) node.
The `staked-access-node-addr` is the address of the staked node this unstaked access node is tethered to.
While the `unstaked` and `staked-access-node-addr` are access node only params, the FlowNodeBuilder initialization needs to be changed for the unstaked node. Hence,  the `staked` variable is defined within scaffold.go. `ExtraFlagsWithFnb` is a new method which allows changing the fnb.Staked flag when parsing the command line arg.